README.md

```diff
@@ -1 +1,5 @@
-Placeholder README for migration of personal repos.
+# sjrk-storyTelling
+
+SJRK Storytelling Project.
+
+UI (so far) for storytelling project.
```

Comment:
Could we expand this README?

Some thoughts:

- Contextual information, such as links to project website
- Overview of what is contained in this repo
- Any instructions to install/run/test what is here